api/managementpb/actions.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto3";
+
+package management;
+
+// This option is used to set the default error response for all Swagger APIs generated from this file.
+// See also header.json.
+// It has to be copy&pasted into each file.
+option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
+  responses: {
+    key: "default"
+    value: {
+      description: "An error response."
+      schema: {
+        json_schema: {
+          ref: "#/definitions/managementpbErrorResponse"
+        }
+      }
+    }
+  }
+};
+option go_package = "managementpb";
+
+import "agentpb/agent.proto";
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+import "google/api/annotations.proto";
+import "protoc-gen-swagger/options/annotations.proto";
+
+// RunActionRequest describes request to run an action.
+message RunActionRequest {
+  // PMM agent instance identifier.
+  string pmm_agent_id = 1;
+  // Service ID for which action is running.
+  string service_id = 2;
+  // Node ID for which action is running.
+  string node_id = 3;
+  // Action name to run.
+  agent.ActionName action_name = 4;
+  // action parameters.
+  repeated string action_params = 5;
```

Comment:
_There_ we should have type-specific parameters for all Actions, both built-in and processes.

Reply:
Let's define which parameters for each action do we need here?

Reply:
Let's not. We have separate tickets for that.

Reply:
Give me the number, please? I have checked all `epic` and didn't find anything. Or what we should do with this? Should we change this API in the context of this task?

Reply:
![image](https://user-images.githubusercontent.com/11512/57634932-e8159a80-75ae-11e9-96ef-99c6f06fae28.png)

Reply:
Ok, so at least at this moment, current params here is OK, as we haven't any description or requirements yet for other commands. So when we will implement those commands we will have ` type-specific parameters for all Actions, both built-in and processes.` here. Or I can make `pt-summary params` here as an example and as a foundation.